uPortal-soffit-renderer/src/main/java/org/apereo/portal/soffit/renderer/SoffitRendererController.java

```diff
@@ -106,72 +121,90 @@
     private String viewsLocation;
     private final Map<ViewTuple,String> availableViews = new HashMap<>();
 
+    private Map<AnnotatedElement,Object> modelAttributes;
+
     protected final Logger logger = LoggerFactory.getLogger(getClass());
 
+    @PostConstruct
+    public void init() {
+        /*
+         * Gather classes & methods that reference @SoffitMoldelAttribute
+         */
+        final Map<AnnotatedElement,Object> map = new HashMap<>();
+        final String[] beanNames = applicationContext.getBeanDefinitionNames();
+        for (String name : beanNames) {
+            final Object bean = applicationContext.getBean(name);
+            final Class clazz = AopUtils.isAopProxy(bean)
```

Comment:
In my experience, `clazz` is a _really_ common variable name for an object of type `Class`;  it's a convention.  http://stackoverflow.com/questions/2529974/why-do-java-programmers-like-to-name-a-variable-clazz